pkg/collector/corechecks/oracle-dbm/oracle.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/datadog-agent/pkg/version"
 	_ "github.com/godror/godror"
+	ttlcache "github.com/jellydator/ttlcache/v2"
```

Comment:
Do you think you could use `github.com/patrickmn/go-cache` instead? Not that I think it's better than this one but it's already in the agent's dependencies and it probably makes sense to try that one before introducing something new.

Reply:
The Agent uses `github.com/patrickmn/go-cache` now.